packages/oscal-react-library/src/components/OSCALParam.tsx

```diff
@@ -0,0 +1,267 @@
+  if (!usage) {
+    return null;
+  }
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Usage</OSCALSectionHeader>
+      <OSCALMarkupMultiLine>{usage}</OSCALMarkupMultiLine>
+    </>
+  );
+};
+
+interface OSCALParamConstraintsProps {
+  constraints: Constraint[] | undefined;
+}
+
+const OSCALParamConstraints: React.FC<OSCALParamConstraintsProps> = ({ constraints }) => {
+  if (!constraints) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Constraints</OSCALSectionHeader>
+      {constraints?.map((constraint) => (
+        <>
+          <OSCALMarkupMultiLine>{constraint.description}</OSCALMarkupMultiLine>
+          <Table>
+            <StyledTableHead>
+              <TableRow>
+                <StyledHeaderTableCell>Expression</StyledHeaderTableCell>
+                <StyledHeaderTableCell>Remarks</StyledHeaderTableCell>
+              </TableRow>
+            </StyledTableHead>
+            <TableBody>
+              {constraint.tests?.map((test) => (
+                <StyledTableRow key={test.expression}>
+                  <TableCell>{test.expression}</TableCell>
+                  <TableCell>{test.remarks}</TableCell>
+                </StyledTableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </>
+      ))}
+    </>
+  );
+};
+
+interface OSCALParamGuidelinesProps {
+  guidelines: Guideline[] | undefined;
+}
+
+const OSCALParamGuidelines: React.FC<OSCALParamGuidelinesProps> = ({ guidelines }) => {
+  if (!guidelines) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Guidelines</OSCALSectionHeader>
+      {guidelines?.map((guideline) => (
+        <OSCALMarkupMultiLine key={guideline.prose}>{guideline.prose}</OSCALMarkupMultiLine>
+      ))}
+    </>
+  );
+};
+
+interface OSCALParamValuesProps {
+  values: string[] | undefined;
+}
+
+const OSCALParamValues: React.FC<OSCALParamValuesProps> = ({ values }) => {
+  if (!values) {
+    return null;
+  }
+
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Values</OSCALSectionHeader>
+      <Typography>{values?.join(", ")}</Typography>
+    </>
+  );
+};
+
+interface OSCALParamLinksProps {
+  links: OSCALLink[] | undefined;
+}
+
+const OSCALParamLinks: React.FC<OSCALParamLinksProps> = ({ links }) => {
+  if (!links) {
+    return null;
+  }
+  return (
+    <>
+      <Divider />
+      <OSCALSectionHeader>Links</OSCALSectionHeader>
+      {links?.map((link) => (
+        <Link key={link.href} href={link.href}>
+          {link.text ?? link.href}
+        </Link>
+      ))}
+    </>
+  );
+};
+
+interface OSCALSelectProps {
+  choice?: string[];
+}
+
+const OSCALSingleSelect: React.FC<OSCALSelectProps> = ({ choice }) => {
+  const [selectValue, setSelectValue] = React.useState("");
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setSelectValue(event.target.value as string);
+  };
```

Comment:
I initially wanted `OSCALSingleSelect` and `OSCALMultiSelect` to be a single `OSCALSelect` that has a `multiple` flag.

This led to a lot of ternary nonsense that I could not quite get to work. If someone has a cleaner way of dealing with a single component, I will be happy to update this.